pkg/clients/edgeconnect/edgeconnect.go

```diff
@@ -43,21 +43,41 @@ type CreateResponse struct {
 	OauthClientSecret          string           `json:"oauthClientSecret"`
 	OauthClientResource        string           `json:"oauthClientResource"`
 	HostPatterns               []string         `json:"hostPatterns"`
+	HostMappings               []HostMapping    `json:"hostMappings"`
 	ManagedByDynatraceOperator bool             `json:"managedByDynatraceOperator,omitempty"`
 }
 
 type Request struct {
-	Name                       string   `json:"name"`
-	OauthClientId              string   `json:"oauthClientId,omitempty"`
-	HostPatterns               []string `json:"hostPatterns"`
-	ManagedByDynatraceOperator bool     `json:"managedByDynatraceOperator,omitempty"`
+	Name                       string        `json:"name"`
+	OauthClientId              string        `json:"oauthClientId,omitempty"`
+	HostPatterns               []string      `json:"hostPatterns"`
+	HostMappings               []HostMapping `json:"hostMappings"`
+	ManagedByDynatraceOperator bool          `json:"managedByDynatraceOperator,omitempty"`
 }
 
-func NewRequest(name string, hostPatterns []string, oauthClientId string) *Request {
+type HostMapping struct {
```

Comment:
This could be just a `map[string]string`

Reply:
Hmmm, yes but then we are not validating the "from" and "to" schema fields. IMO is better to have the struct

Reply:
Fixed [here](https://github.com/Dynatrace/dynatrace-operator/pull/3332/commits/11075d6cd8e6e211a774059eae3c459c68864c68)
I refactored quite a bit, but I think it is better this way. Now we have an edgeConnect.HostMappings() function the same way as we have the HostPatterns() one.